app/customer.py

```diff
@@ -0,0 +1,47 @@
+from math import sqrt
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(self,
+                 name: str,
+                 product_cart: dict,
+                 location: list[int],
+                 money: float,
+                 car: dict) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = Car(car["brand"], car["fuel_consumption"])
+
+    def distance_to(self, shop_location: list[int]) -> float:
+        return sqrt(
+            (self.location[0] - shop_location[0]) ** 2
+            + (self.location[1] - shop_location[1]) ** 2
+        )
+
+    def calculate_total_trip_cost(self,
+                                  shop: Shop,
+                                  fuel_price: float) -> float:
+        distance = self.distance_to(shop.location)
+        fuel_cost_to_shop = self.car.calculate_fuel_cost(distance, fuel_price)
+        fuel_cost_to_home = self.car.calculate_fuel_cost(distance, fuel_price)
```

Comment:
This is a code duplication issue. You are calculating the fuel cost to the shop and back home separately, but since the distance is the same, you can calculate it once and multiply by two to avoid code duplication. This aligns with the checklist item: 'Don't Repeat Yourself'.